modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml

```diff
@@ -42,6 +42,7 @@ assignments:
   exclude:
     types:
       config:
+        config_snapshot: config_snapshot
```

Comment:
The only thing is as far as I know this config file was recently removed in https://github.com/systemseed/falcon/pull/53, because it conflicted on installation. I'll leave a note there.